test/integration/log_system.cc

```diff
@@ -445,12 +445,14 @@ TEST_F(LogSystemTest, LogPaths)
   }
 
   // Check state.tlog is no longer stored to path specified in SDF
-  EXPECT_FALSE(common::exists(common::joinPaths(this->logDir,
-      "state.tlog")));
+  auto stateFile = common::joinPaths(this->logDir, "state.tlog");
+  EXPECT_FALSE(common::exists(stateFile)) << stateFile;
 #ifndef __APPLE__
   EXPECT_EQ(0, entryCount(this->logDir));
 #endif
 
+  return;
```

Comment:
Is this `return` supposed to be here?

Reply:
I had pushed that by mistake, it was removed in d693bb636e09754ec16c782ff22d9dc6b93282ad